### DISCUSSION TITLE
Feature request: ability to specify topic/member to base map positioning on

### DISCUSSION THREAD

**Discussion by russkel** [foxglove/community#664] [Q&A]
At the moment it seems we are unable to set the topic to base the positioning of a map on.

In addition adding the type `geographic_msgs/msg/GeoPoint` to the Location schema so the map could be based off GeoPoses and the like would further complete the functionality.

**Comment by defunctzombie** [comment]
> At the moment it seems we are unable to set the topic to base the positioning of a map on.

Have you tried picking a `Follow topic` in the panel settings?

**Reply by russkel** [reply]
I have not. I do not remember seeing that option, but that may have been because we had nothing that implemented the `Location` schema in the mcap? If this functionality is there then I apologise, but my request for `GeoPoint` support would be very appreciated :).

**Reply by defunctzombie** [reply]
You can use a message converter extension (https://foxglove.dev/docs/studio/extensions/getting-started#message-converters) to convert the `geographic_msgs/msg/GeoPoint` into a `sensor_msgs/NavSatFix` or a simpler `foxglove.LocationFix` until we have native support for it in the panel.
